pkg/detectlicense/licenses.go

```diff
@@ -21,20 +21,25 @@ type License struct {
 var (
 	Proprietary = License{Name: "proprietary"}
 
-	PublicDomain = License{Name: "public domain"}
-
-	Apache2 = License{Name: "Apache License 2.0", NoticeFile: true, Url: "https://opensource.org/licenses/Apache-2.0"}
-	BSD1    = License{Name: "1-clause BSD license", Url: "https://opensource.org/licenses/BSD-1-Clause"}
-	BSD2    = License{Name: "2-clause BSD license", Url: "https://opensource.org/licenses/BSD-2-Clause"}
-	BSD3    = License{Name: "3-clause BSD license", Url: "https://opensource.org/licenses/BSD-3-Clause"}
-	ISC     = License{Name: "ISC license", Url: "https://opensource.org/licenses/ISC"}
-	MIT     = License{Name: "MIT license", Url: "https://opensource.org/licenses/MIT"}
-	MPL2    = License{Name: "Mozilla Public License 2.0", NoticeFile: true,
-		WeakCopyleft: true, Url: "https://opensource.org/licenses/MPL-2.0"}
+	PublicDomain = License{Name: "Public domain"}
```

Comment:
"public" is not a proper noun here, it is properly lowercase

Reply:
Given it's being used as a name, shouldn't it be capitalized?

It's being used to generate markdown like this
`Emissary-ingress Python code incorporates Free and Open Source software under the following licenses:

* [2-clause BSD license](https://opensource.org/licenses/BSD-2-Clause)
* [3-clause BSD license](https://opensource.org/licenses/BSD-3-Clause)
* [Apache License 2.0](https://opensource.org/licenses/Apache-2.0)
* [GNU General Public License Version 3](https://opensource.org/licenses/GPL-3.0)
* [GNU Lesser General Public License Version 2.1](https://spdx.org/licenses/LGPL-2.1-or-later.html)
* [ISC license](https://opensource.org/licenses/ISC)
* [MIT license](https://opensource.org/licenses/MIT)
* [Mozilla Public License 2.0](https://opensource.org/licenses/MPL-2.0)
* [Python Software Foundation license](https://spdx.org/licenses/PSF-2.0.html)
* [Unicode License Agreement for Data Files and Software (2015)](https://spdx.org/licenses/Unicode-DFS-2015.html)
* public domain`

And that will be published to the [website](https://1506.netlify-preview.datawire.io/docs/emissary/2.1/licenses/)

I can just capitalize it when Markdown is generated but I changed it because of the assumptions that it was used as a name

Let me know if you still think it should be changed and I can address this.

Reply:
@LukeShu please let me know if you agree with my reasoning there or not.